cloud-haven-react-app/src/components/UserProfile/UserProfile.js

```diff
@@ -0,0 +1,162 @@
+import React from "react";
```

Comment:
are these changes in the right pull request?

Reply:
I got a little confused and thought we were working on the entire data management popup at first so we created this branch off the user-profile branch thinking we would need it. Turns out we didn't need it for this so we will just pull from main again on this branch once the user-profile is approved and merged in.